fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -555,7 +555,7 @@ recoverPartitionStatement
 // ------------------------------------------- View Statement ----------------------------------------------------------
 
 createViewStatement
-    : CREATE VIEW (IF NOT EXISTS)? qualifiedName
+    : CREATE (OR REPLACE)? VIEW (IF NOT EXISTS)? qualifiedName
```

Comment:
Should we support "create or replace view if not exist"?
What does that mean?
I think create or replace is that create if not exist and replace if not exist.

Reply:
fixed. it's conflicted options, we don't support it.